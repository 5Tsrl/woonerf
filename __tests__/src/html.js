/* globals describe, expect, it */

import html from '../../src/html'

describe('html', () => {
  it('should return an HTML string with a given title', () => {
    const title = 'Unique Title'
    const str = html({title})
    expect(str.indexOf(title) !== -1).toBeTruthy()
    expect(str).toMatchSnapshot()
  })

  it('should return an HTML string with a given staticHost', () => {
    const staticHost = 'https://fakeHost.com/'
    const title = 'Unique Title'
    const str = html({staticHost, title})
    expect(str.indexOf(staticHost) !== -1).toBeTruthy()
    expect(str).toMatchSnapshot()
  })
})
