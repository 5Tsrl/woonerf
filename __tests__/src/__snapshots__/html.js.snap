exports[`html should return an HTML string with a given title 1`] = `
"
  <!DOCTYPE html>
  <html>
    <head>
      <meta charset=\"utf-8\">
      <meta name=\"viewport\" content=\"width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0\">
      <link href=\"assets/index.css\" rel=\"stylesheet\">

      <title>Unique Title</title>
    </head>
    <body>
      <div id=\"root\"></div>
      <script src=\"assets/index.js\"></script>
    </body>
  </html>
  "
`;
